BaseFilter/sections/adservers.txt

```diff
@@ -122,6 +122,7 @@
 ||ap.frarychazan.com^
 ||trestonclop.com^$all
 ||guroshied.com^$all
+||cdn.b2.ai^
```

Comment:
Need a real example of this as I asked [here](https://github.com/AdguardTeam/AdguardFilters/issues/173253#issuecomment-1956111062).
Btw, wrong file. You should add it to the `antiadblock.txt` with the issue comment.

Reply:
Add the rule next to `! gplinks -` detection in the file like this.
```adblock
! b2.ai - adblock detection
||cdn.b2.ai^
!
```